Gamma_Analysis.py

```diff
@@ -127,6 +127,10 @@ def isotope_concentration(isotope, reference, sample_activity,
         conversion = 1
     not_in_dirt = ['Cs134', 'Cs137', 'Co60', 'Pb210']
     ref_specific_activity = reference_activity[0] / reference.mass
+
+    if type(reference) is ref.base_reference_class():
+    elif type(reference) is ref.petri_reference_class():
```

Comment:
Hmm I was afraid of the same inheritance issue with the use of the `isinstance()` function, but the way you put it is a clever way to simplify the code! The updated code is now live.